tests/acceptance/WooCommerceDynamicSegmentsCest.php

```diff
@@ -101,6 +109,23 @@ public function addCustomerOnlyToCategorySegment(\AcceptanceTester $i) {
     $i->see('0', $productSegmentRow . " [data-colname='Number of subscribers']");
   }
 
+  public function checkThatCustomersAreAddedToNumberOfOrdersSegment(\AcceptanceTester $i) {
```

Comment:
I can see you are checking here that the segment filters subscribers correctly. That is great and we definitely need that. But why did you decide to create the test using a DB call? Shall we create the segment in the UI? Just to check that the creating works?

Reply:
I just followed the pattern that we are already using for the acceptance tests for the other WooCommerce related segments. I don't know what was the reasoning behind only testing that subscribing works correctly in the acceptance tests and not including a test to make sure that creating the segment works. I will look into adding a test covering the creation of the segment, and if it is straightforward, I will add a new commit. I just suggest that we don't hold this PR because of the lack of this test as tomorrow is my last day before my sabbatical, so today is basically the last day that I have to finish this while someone else from the team is around to approve it.

Reply:
Just a quick update that I found that Rosta added new acceptance tests for creating and editing the other WooCommerce segments earlier this week (https://github.com/mailpoet/mailpoet/pull/3443/files#diff-dea08370546d62b9522564eef934ea96d12212880401ee358d09445b316b08d2R316), so I will use those to create a similar test for this segment.